### PR TITLE
Add `Edger8r::search_path()` method

### PR DESCRIPTION
**Pull request by nick-mobilecoin**
The `Edger8r` now supports having search paths specified for importing
dependent edl files

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #245** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/245" target="_blank">Graphite</a> 
    * **PR #251** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/251" target="_blank">Graphite</a> 
      * **PR #253** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/253" target="_blank">Graphite</a> 
        * **PR #254** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/254" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/254?utm_source=stack-comment).

File: sdk-tools/src/edger8r.rs [hunk 1]
```diff
@@ -383,4 +429,65 @@ mod tests {
             assert!(!file.exists())
         }
     }
+
+    #[test]
+    fn dependent_edl_in_search_paths() {
```
Comment (nick-mobilecoin) [inline sdk-tools/src/edger8r.rs:434]: Up for suggestions on how to make this more consise.  Was on the fence if functionalizing more would be clearer or just another level of indirection for the test

Comment (NotGyro) [reply sdk-tools/src/edger8r.rs:428]: Long names aren't as much of a problem for tests, specifically, I think. That may just be a stylistic opinion of mine, though.

**Review by NotGyro — APPROVED** [review]
This looks good to me.

File: sdk-tools/src/edger8r.rs (hunk 1 above)
Comment (nick-mobilecoin) [reply sdk-tools/src/edger8r.rs:428]: Sorry, I knew what I was thinking but failed to fully communicate.
I think the body of the test function is larger than I would like.
As for naming test functions, I agree that tests have more leniency here, and I need to work on improving naming tests I write.

**Review by varsha888 — APPROVED** [review]

**Pull request merged**